core/blockchain.go

```diff
@@ -43,7 +43,7 @@ import (
 	"github.com/ava-labs/coreth/consensus/misc/eip4844"
 	"github.com/ava-labs/coreth/core/state"
 	"github.com/ava-labs/coreth/core/state/snapshot"
-	"github.com/ava-labs/coreth/core/types"
+	customtypes "github.com/ava-labs/coreth/core/types"
```

Comment:
I think this is going to be error-prone / confusing.
I'm okay if we do a fast-follow to move the package to at least a different name.

Reply:
Agreed, some people will forget to use the customtypes alias which will get confusing.
Perhaps we can have a plugin/evm/customtypes or plugin/evm/core/customtypes package?

Reply:
I'll start a fast-follow PR to move everything except the `avalanchego` shim. Doing it here would break the KISS requirements of this PR chain.

Reply:
Moving in #877